jenkins/deployment/community/cortx-community-deploy.groovy

```diff
@@ -15,156 +14,170 @@ pipeline {
         ansiColor('xterm')
     }
 
-
     parameters {
         string(name: 'CORTX_RE_BRANCH', defaultValue: 'main', description: 'Branch or GitHash for CORTX Cluster scripts', trim: true)
-        string(name: 'CORTX_RE_REPO', defaultValue: 'https://github.com/Seagate/cortx-re/', description: 'Repository for CORTX Cluster scripts', trim: true)
+        string(name: 'CORTX_RE_REPO', defaultValue: 'https://github.com/Seagate/cortx-re', description: 'Repository for CORTX Cluster scripts', trim: true)
         string(name: 'CORTX_TAG', defaultValue: 'main', description: 'Branch or GitHash for generaing CORTX container images', trim: true)
         string(name: 'OS_VERSION', defaultValue: 'CentOS 7.9.2009 x86_64', description: 'Operating system version', trim: true)
         string(name: 'REGION', defaultValue: 'ap-south-1', description: 'AWS region', trim: true)
-        string(name: 'KEY_NAME', defaultValue: 'automation-key', description: 'Key name', trim: true)
```

Comment:
Please use `automation-key` as the key name. Basically we want to use a different key than the documented at https://github.com/Seagate/cortx-re/tree/main/solutions/community-deploy/cloud/AWS#note so that Jenkins pipeline execution will not conflict with manual execution.

Reply:
fixed